.github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml

```diff
@@ -60,7 +61,7 @@ jobs:
     name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
     strategy:
       matrix:
-        job_name: ["beam_PostCommit_XVR_GoUsingJava_Dataflow"]
+        job_name: ["beam_PostCommit_XVR_GoUsingJava_Dataflow(DISABLED)"]
```

Comment:
LMK if there is a smarter way to disable a suite.

Reply:
thanks for taking care of this, commented below.